examples/list_resources_example.py

```diff
@@ -0,0 +1,8 @@
+from odp_sdk.client import OdpClient
+
+client = OdpClient()
+
+catalog_client = client.catalog
```

Comment:
No need to assign a property to a new variable

Reply:
What do you mean here? Can you show an example?